sdk/identity/identity/test/public/node/multiTenantAuthentication.spec.ts

```diff
@@ -6,8 +6,8 @@ import { msalNodeTestSetup } from "../../node/msalNodeTestSetup.js";
 import type { Recorder } from "@azure-tools/test-recorder";
 import { env } from "@azure-tools/test-recorder";
 import { createHttpHeaders, createPipelineRequest } from "@azure/core-rest-pipeline";
-import { ClientSecretCredential } from "../../../src/credentials/clientSecretCredential.js";
-import { IdentityClient } from "../../../src/client/identityClient.js";
+import { ClientSecretCredential } from "../../../dist/esm/credentials/clientSecretCredential.js";
+import { IdentityClient } from "../../../dist/esm/client/identityClient.js";
```

Comment:
@KarishmaGhiya This is a public test and I wonder why are these imported from internal code.

Reply:
It shouldn't have been imported from internal. The set up is wrong

Reply:
Let's track the fix to this outside this PR since it is unrelated.